packages/devtools-source-map/src/source-map.js

```diff
@@ -165,11 +171,31 @@ async function getOriginalLocation(location: Location): Promise<Location> {
     return location;
   }
 
-  const { source: sourceUrl, line, column } = map.originalPositionFor({
+  // First check for an exact match
+  let match = map.originalPositionFor({
     line: location.line,
     column: location.column == null ? 0 : location.column
   });
 
+  // If there is not an exact match, look for a match with a bias at the
+  // current location and then on subsequent lines
+  if (search) {
+    let line = location.line;
+    let column = location.column == null ? 0 : location.column;
+
+    while (!match.sourceUrl) {
+      match = map.originalPositionFor({
+        line,
+        column,
+        bias: SourceMapConsumer[search]
+      });
+
+      line = search == "LEAST_UPPER_BOUND" ? line++ : line--;
```

Comment:
I think assigning and doing `line++` on the same line is a little confusing. Maybe

```
line += search == "LEAST_UPPER_BOUND" ? 1 : -1;
```
?